### PR TITLE
fix: Do not register fastify-websocket if it is already registered

### PR DESCRIPTION
**Pull request by acro5piano**
Hi, thank you for the great library.

This PR fixes https://github.com/mercurius-js/mercurius/issues/369, which enables `mercurius` to work with `fastify-websocket`.

As @mcollina suggested on https://github.com/mercurius-js/mercurius/issues/369#issuecomment-751381695, I've added a check if `fastify-websocket` is already registered or not.

- My determination is that checking the decorator `fastify.websocketServer` is `undefined` or not. I think there should be a method to check a plugin registered or not, but this is what I know about Fastify.
- I've added a simple test for coverage. I've confirmed subscription is working even if `fastify-websocket` is already registered on my machine, but I'll happy to add more tests if needed.
 
Thanks!

### CODE REVIEW

File: test/subscription.js [hunk 1]
```diff
@@ -1751,3 +1752,33 @@ test('`withFilter` tool works with async filters', t => {
     })
   })
 })
+
+test('subscription server does not register fastify-websocket if already registered', t => {
```
Comment (mcollina) [inline test/subscription.js:1756]: could you please expand this test by using the graphql subscription and adding another non-graphql websocket route?

Comment (acro5piano) [reply test/subscription.js:1756]: ok!

Comment (acro5piano) [reply test/subscription.js]: @mcollina I've added the test, so could you check it again? Thank you for your review!

**Review by mcollina — APPROVED** [review]
lgtm, good work!

**Comment by acro5piano** [comment]
@mcollina Thank you!

**Pull request merged**